etc/config/c.amazon.properties

```diff
@@ -236,6 +236,11 @@ compiler.cclang_assertions_trunk.demangler=/opt/compiler-explorer/gcc-snapshot/b
 compiler.cclang_assertions_trunk.objdumper=/opt/compiler-explorer/gcc-snapshot/bin/objdump
 compiler.cclang_assertions_trunk.semver=(assertions trunk)
 compiler.cclang_assertions_trunk.options=--gcc-toolchain=/opt/compiler-explorer/gcc-9.2.0
+compiler.cclang_dang.exe=/opt/compiler-explorer/clang-dang-main/bin/clang
+compiler.cclang_dang.demangler=/opt/compiler-explorer/gcc-snapshot/bin/c++filt
+compiler.cclang_dang.objdumper=/opt/compiler-explorer/gcc-snapshot/bin/objdump
+compiler.cclang_dang.semver=(thephd.dev)
+compiler.cclang_dang.options=-std=c2x
```

Comment:
no --gcc-toolchain needed?

Reply:
Uhh, maybe...? I hope! At least I didn't have to specify it when I was developing locally, but maybe I should go back and add that in...

Reply:
you probably have a local gcc and binutils installed, but our are pretty old and we like to have it up to date

Reply:
Ahem. Sorry, added it!